cypress/features/regression/dialogFullScreen.feature

```diff
@@ -1,7 +1,7 @@
 Feature: Dialog Full Screen component
```

Comment:
Should we add test related to event in Action Tab ?

Reply:
updated

Reply:
added